### PR TITLE
Packaging and documentation updates

### PR DESCRIPTION
**Pull request by saaros**
* "make deb" now builds a python 3 package
* rpm package obsoletes kafkajournalpump
* basic test case that just imports journalpump
* added notes to NEWS

### CODE REVIEW

File: journalpump/journalpump.py [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# Copyright 2016, Aiven, https://aiven.io/
+# Copyright 2015, Aiven, https://aiven.io/
```
Comment (Ormod) [inline journalpump/journalpump.py]: Lets just leave this as 2016

File: test/test_basic.py [hunk 2]
```diff
@@ -0,0 +1,10 @@
+from journalpump import journalpump
+from tempfile import NamedTemporaryFile
+import json
+
+
+def test_basic():
```
Comment (Ormod) [inline test/test_basic.py]: I think all this and more are covered by the unittests in the file test_journalpump.py

File: debian/changelog.in [hunk 3]
```diff
@@ -1,10 +1,4 @@
-journalpump (2.0.0) unstable; urgency=low
```
Comment (Ormod) [inline debian/changelog.in]: Hmm, why remove 2.0.0 from here when its kept in .spec?

**Comment by Ormod** [comment]
And travis tests need to pass as well.

File: journalpump/journalpump.py (hunk 1 above)
Comment (saaros) [inline journalpump/journalpump.py]: Copyright year is supposed to include the date of first publication, updating it every year is pointless (or eve counter productive).  Also, none of the other copyright claims were updated to match this.

File: debian/changelog.in (hunk 3 above)
Comment (saaros) [inline debian/changelog.in]: debian package version is autogenerated, and the current tags produce a smaller version than 2.0.0 which in turn causes debian package build to fail.

**Comment by saaros** [comment]
Missing flake8 dependency and unused machine-id reading caused tests to fail in travis. fixed.

**Pull request merged**